### PR TITLE
Added 404 Page

### LINKED ISSUE #28: Add 404 Page
image

we need a 404 page, which has a button to go back to home screen.
Page should look good.

### PR DESCRIPTION
**Pull request by harshmetkel24**
I have created a 404 page. Also if someone wants to visit Netlify's docs they can do so directly by clicking on "page not found".
If changes are needed let me know.

### CODE REVIEW

**Review by atoughman — CHANGES_REQUESTED** [review]
In React, even if you import a CSS file seprately in one component, still it's styles are applied globally. Because of this, your CSS file has broken some of the styling.

Can you please give components some specific classNames and then apply styles only to them.

**Comment by amanRecreates** [comment]
Hi @harshmetkel24 , could you please look into above fix.

**Comment by harshmetkel24** [comment]
> 
> 
> Hi @harshmetkel24 , could you please look into above fix.

Yeah i will fix this asap

**Comment by harshmetkel24** [comment]
> 
> 
> If the problem is not clear, let me know I will try to explain in other way.

No I understood what you are trying to say, I guess my styles are creating one contanier at home route also. I will change classnames to my style file so that it doesn't contradicts to existing styles. 

Thanks for pointing out this.

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Code-Type-WebApp</title>
```
Comment (atoughman) [inline public/index.html:27]: Different Issues needs to be addressed in different PR's.
Somebody has fixed this in previous PR. You can take a fresh pull and ignore this change.

File: src/Components/Error/error.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
+
+*{
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+
+body{
+	background-color: whitesmoke;
+}
```
Comment (atoughman) [inline src/Components/Error/error.css:10]: You can delete this. Because it is already present in global stylesheet.

File: src/Components/Error/error.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+.hr-2{
+	height: 1px;
+	background-color: black;
+	width: 100%;
+	margin: 2px auto;
+	border-radius: 5px;
+}
+
+.body-2{
+		text-align: justify;
```
Comment (atoughman) [inline src/Components/Error/error.css:69]: Fix indentation.

**Review by atoughman — CHANGES_REQUESTED** [review]

**Comment by atoughman** [comment]
Hi @harshmetkel24 , It looks like I didn't make issue clear. Sorry for that. We don't need exact 404 page as shown by default. Instead a new one is required.
I think I should make the design of 404 Page and then post it in the issue.
Till then you can pause work on this.

File: public/index.html (hunk 1 above)
Comment (harshmetkel24) [reply public/index.html:27]: Okay got it.
This are some of my initial contributions, so making such a mistakes.
Thanks for the pinpointing my mistakes :)

File: src/Components/Error/error.css (hunk 3 above)
Comment (harshmetkel24) [reply src/Components/Error/error.css:69]: Yeah I will apply indentation of tab size of 4-spaces to it.

File: src/Components/Error/error.css (hunk 2 above)
Comment (harshmetkel24) [reply src/Components/Error/error.css:10]: Hmm got it.

**Comment by harshmetkel24** [comment]
> Hi @harshmetkel24 , It looks like I didn't make issue clear. Sorry for that. We don't need exact 404 page as shown by default. Instead a new one is required.
> I think I should make the design of 404 Page and then post it in the issue.
> Till then you can pause work on this.

Okay no problem!
